### PR TITLE
sessionctx: make tidb_max_chunk_size as a global variable

### PR DESCRIPTION
**Pull request by jackysp**
The default tidb_max_chunk_size (1024) slows down the performance of OLTP workload, like TPC-C.

| tidb_max_chunk_size |  tpmC |
| - | - |
| 1024 | 10549 |
| 1 | 43515 |

One of the biggest malloc is the following one:

image

This pr makes tidb_max_chunk_size as a global variable for turning the performance easily.

PTAL @XuHuaiyu @zz-jason

### CODE REVIEW

**Comment by coocood** [comment]
@jackysp 
Is it possible to set this variable to 0?

File: sessionctx/variable/sysvar.go [hunk 1]
```diff
@@ -624,7 +624,7 @@ var defaultSysVars = []*SysVar{
 	{ScopeSession, TiDBBatchDelete, boolToIntStr(DefBatchDelete)},
 	{ScopeSession, TiDBDMLBatchSize, strconv.Itoa(DefDMLBatchSize)},
 	{ScopeSession, TiDBCurrentTS, strconv.Itoa(DefCurretTS)},
-	{ScopeSession, TiDBMaxChunkSize, strconv.Itoa(DefMaxChunkSize)},
+	{ScopeGlobal | ScopeSession, TiDBMaxChunkSize, strconv.Itoa(DefMaxChunkSize)},
```
[the remark this answers is not in the document]
Comment (jackysp) [reply sessionctx/variable/sysvar.go]: Maybe we can do it in another pr.

**Comment by jackysp** [comment]
It could be, but the value will convert to the default max chunk size. But it is still shown as 0, if you try to show this variable. I think it is a bug. @coocood

**Comment by coocood** [comment]
LGTM

**Pull request merged**